lib/tasks/report.rake

```diff
@@ -1,9 +1,10 @@
 namespace :report do
   desc "subdomain analytics and report tasks"
-  
+
   task :send_analytics_report => [:environment] do 
     p "starting send_analytics_report task  @ #{Time.now}"
-    Subdomain.where.not(analytics_report_frequency: Subdomain::REPORT_FREQUENCY_MAPPING[:never]).each do |subdomain|
+    subdomains = Subdomain.all_with_public_schema
```

Comment:
@Ayon95 nice! Before we were working with an activerecord collection, but with your change its just a simple array (because we shovel on an initialized subdomain) so we are filtering out any subdomain that is not reporting analytics